### PR TITLE
Adding Metrics Based Argo Rollouts

### PR DESCRIPTION
**Pull request by kdubois**

### CODE REVIEW

File: scripts/argorollouts/clean.sh
Comment (lordofthejars) [inline scripts/argorollouts/clean.sh:1]: Could you create a new file called `clean-metrics.sh` so the old one still works?

File: scripts/argorollouts/deploy.sh
Comment (lordofthejars) [inline scripts/argorollouts/deploy.sh:1]: Can you rename to `deploy-metrics.sh` so we keep the old one too?

File: scripts/argorollouts/rolling-to-v2.sh
Comment (lordofthejars) [inline scripts/argorollouts/rolling-to-v2.sh:1]: Can you rename it so we have the original one too?

**Review by lordofthejars — CHANGES_REQUESTED** [review]

File: scripts/argorollouts/clean.sh
Comment (kdubois) [reply scripts/argorollouts/clean.sh:1]: I created a new folder, that makes it less confusing I think

**Comment by kdubois** [comment]
@lordofthejars can you review again?  If you merge perhaps use squash because there are a lot of commits :D

**Review by lordofthejars — APPROVED** [review]

**Pull request merged**